cray_product_catalog/constants.py

```diff
@@ -35,3 +35,4 @@
 COMPONENT_MANIFESTS = 'manifests'
 CONFIG_MAP_FIELDS = {'configuration', 'images', 'recipes'}
 PRODUCT_CM_FIELDS = {'component_versions'}
+PRODUCT_CATALOG_CONFIG_MAP_LABEL = 'type='+PRODUCT_CATALOG_CONFIG_MAP_NAME
```

Comment:
I was thinking something like this and then use them directly :
`
PRODUCT_CATALOG_CONFIG_MAP_LABEL_KEY = 'type'
PRODUCT_CATALOG_CONFIG_MAP_LABEL = {PRODUCT_CATALOG_CONFIG_MAP_LABEL_KEY: PRODUCT_CATALOG_CONFIG_MAP_NAME}

PRODUCT_CATALOG_CONFIG_MAP_LABEL_STR = "{0}={1}".format(PRODUCT_CATALOG_CONFIG_MAP_LABEL_KEY, PRODUCT_CATALOG_CONFIG_MAP_NAME)
`